modules/service/trac-price-service.js

```diff
@@ -1,47 +1,25 @@
 const axios = require('axios');
 const constants = require('../constants');
 
-const coinGeckoLink = 'https://api.coingecko.com/api/v3/simple/price?ids=origintrail&vs_currencies=eth';
+const coinGeckoLink = '';
 
 class TracPriceService {
```

Comment:
This is now more coin-gecko-service as it's purpose is to fetch trac price from coin gecko link.